packages/h5wasm/src/h5wasm-api.ts

```diff
@@ -154,7 +148,6 @@ export class H5WasmApi extends ProviderApi {
         ...baseGroup,
         children: h5wEntity.keys().map((childName) => {
           const h5wChild = h5wEntity.get(childName);
-          assertNonNull(h5wChild);
```

Comment:
The fallback is already present below so only the removal of assertions are needed